From f1f08727d891361a3bcf8387e15bbe9e9b4230fd Mon Sep 17 00:00:00 2001
From: Sai Sree Kartheek Adivi <s-adivi@ti.com>
Date: Wed, 25 May 2022 09:59:25 +0530
Subject: [PATCH 1/3] main.qml: Handle 3D button click

When 3D button is clicked, exit with return code 0x3D (61).
The return value should be handled by the launch script to launch 3D
Demo or Matrix-GUI.

Signed-off-by: Sai Sree Kartheek Adivi <s-adivi@ti.com>
---
 main.qml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/main.qml b/main.qml
index b0a706a..c44017f 100644
--- a/main.qml
+++ b/main.qml
@@ -613,7 +613,8 @@ Window {
                     }
                     onClicked: {
                         renderfill.color = "#C6C6C6"
-                        textupdate.text = "3D Demo Selected (Coming Soon)"
+                        textupdate.text = "Launching 3D Demo .."
+                        Qt.exit(0x3D)
                     }
                     onReleased: {
                         renderfill.color = "transparent"
-- 
2.24.3

